api/src/main/java/org/openmrs/module/locationbasedaccess/LocationBasedAccessActivator.java

```diff
@@ -46,6 +52,15 @@ public void willStart() {
      * @see ModuleActivator#started()
      */
     public void started() {
+        String locationAttributeUuid = Context.getAdministrationService().getGlobalProperty("locationbasedaccess.locationAttributeUuid");
+        PersonAttributeType personAttributeType = new PersonAttributeType();
+        if (StringUtils.isNotBlank(locationAttributeUuid)) {
+            personAttributeType.setUuid(locationAttributeUuid);
+        }
+        personAttributeType.setName(this.LOCATION_ATTRIBUTE_TYPE_NAME);
```

Comment:
Why are you using this.?

Reply:
Actually coming to think about it, since the user will have to configure the registration form and supply a uuid, let us let them create it for themselves. Instead of us doing it in code here. I would like our initial code to be as simple and small as possible.

Reply:
Yes, they can give the UUID for themselves, but We need to define the attribute name inside the implementation to track the future implementations. Am I right? 
I need to get the personAttribute using `person.getAttribute(attributeName)` in other classes.

Reply:
What do you mean by future implementations?
Is there any implementation which cannot just use a uuid value for the global property to specify which person attribute type stores the location?